src/components/Tutorials/subComps/ControlButtons.jsx

```diff
@@ -2,61 +2,101 @@ import React from "react";
-							onClick={() => {
-								setCurrentStep(currentStep + 1);
-								window.scrollTo(0, 0);
-							}}
-							style={{ float: "right" }}>
-							Next
-						</Button>
-					)}
-					{currentStep === stepsData.length - 1 && (
-						<Button
-							type="primary"
-							onClick={() => {
-								<Snackbar
-									anchorOrigin={{
-										vertical: "bottom",
-										horizontal: "left",
-									}}
-									open={true}
-									autoHideDuration={6000}
-									message="tutorial complete"
-								/>;
-								window.scrollTo(0, 0);
-							}}
-							style={{ float: "right" }}>
-							Finish
-						</Button>
-					)}
-				</div>
-			</Grid>
-		);
-	} else return null;
+const useStyles = makeStyles(theme => ({
+  container: {
+    display: "flex",
+    justifyContent: "space-between",
+    [theme.breakpoints.down("sm")]: {
+      flexDirection: "column",
+      alignItems: "center"
+    }
+  },
+  rightButtonsGroup: {
+    display: "flex",
+    gap: "5px"
+  },
+  prevButton: {
+    borderWidth: "2px",
+    "&:hover": {
+      borderWidth: "2px"
+    }
+  }
+}));
+
+const ControlButtons = ({
+  currentStep,
+  setCurrentStep,
+  stepsData,
+  hide,
+  setStepData
+}) => {
+  const classes = useStyles();
+  if (!hide && stepsData) {
+    return (
+      <Grid>
+        <Box className={classes.container}>
+          <Button
+            color="primary"
+            variant="outlined"
+            data-testid="previousStepButton"
+            onClick={() => {
+              setCurrentStep(currentStep - 1);
+              window.scrollTo(0, 0);
+            }}
+            disabled={currentStep === 0}
+            className={classes.prevButton}
+          >
+            Previous
+          </Button>
+          <Box className={classes.rightButtonsGroup}>
+            <Button
+              variant="contained"
+              color="primary"
+              type="primary"
+              data-testid="nextStepButton"
+              onClick={() => {
+                setCurrentStep(currentStep + 1);
+                window.scrollTo(0, 0);
+              }}
+              disabled={currentStep >= stepsData.length - 1}
+            >
+              Next
+            </Button>
+            <Button
+              type="primary"
+              onClick={() => {
+                <Snackbar
+                  anchorOrigin={{
+                    vertical: "bottom",
+                    horizontal: "left"
+                  }}
+                  open={true}
+                  autoHideDuration={6000}
+                  message="tutorial complete"
+                />;
+                window.scrollTo(0, 0);
+                setStepData(prevSteps =>
+                  prevSteps.map((step, index) =>
+                    index === currentStep
+                      ? { ...step, completed: !step.completed }
+                      : step
+                  )
+                );
+              }}
+              // style={{ float: "right" }}
```

Comment:
remove this comment

Reply:
I'll change that in the next pr, is that okay?